src/components/MessageBox.js

```diff
@@ -11,55 +12,51 @@ const StyleUtils = require('../utils/Style');
 
 class MessageBox extends React.Component {
   static propTypes = {
-    children: PropTypes.node,
+    button: PropTypes.node,
     color: PropTypes.string,
     icon: PropTypes.string,
+    isSmall: PropTypes.bool,
+    message: PropTypes.node,
     styles: PropTypes.object,
     theme: themeShape,
     title: PropTypes.string
   };
 
-  state = {
-    isOpen: true
-  };
-
-  _toggleMessageBox = () => {
-    this.setState({
-      isOpen: !this.state.isOpen
-    });
-  };
-
   render () {
     const theme = StyleUtils.mergeTheme(this.props.theme);
     const styles = this.styles(theme);
 
     return (
       <div className='mx-message-box' style={styles.component}>
-        <div onClick={this.props.children ? this._toggleMessageBox : () => {}} style={styles.header}>
-          <div style={styles.leftHeader}>
-            <Icon
-              size={20}
-              style={Object.assign({}, styles.icon, { marginRight: theme.Spacing.SMALL })}
-              type={this.props.icon}
-            />
-            <div style={styles.title}>{this.props.title}</div>
+        <div style={styles.alertbar} />
+        <div style={styles.messageSection}>
+          <div style={styles.messageBody}>
+            <div>
+              <Icon
+                size={20}
+                style={{ ...styles.icon, ...{ marginRight: theme.Spacing.SMALL } }}
```

Comment:
Hmm, now that I look at this closer, there is no conditional for applying `marginRight` here. It would probably be better if we move `marginRight` inside the `styles.icon` object directly to simplify things..

```javascript
      style={styles.icon}

      icon: {
        fill: theme.Colors.GRAY_900,
        marginRight: theme.Spacing.SMALL
      },

Reply:
good call, I just pushed that change